reference/employer/openapi.yaml

```diff
@@ -1029,13 +1029,25 @@ paths:
                       description: The url to the official logo of the payroll provider.
                     mfa_required:
                       type: boolean
-                      description: whether MFA is required for the provider.
+                      description: 'whether MFA is required for the provider.'
                     primary_color:
                       type: string
                       description: The hex code for the primary color of the payroll provider.
                     manual:
                       type: boolean
-                      description: Whether the Finch integration with this provider uses the Assisted Connect Flow by default.
+                      description: '[DEPRECATED] Whether the Finch integration with this provider uses the Assisted Connect Flow by default. This field is now deprecated. Please check for a `type` of `assisted` in the `authentication_methods` field instead.'
+                    authentication_methods:
+                      type: object
+                      description: The list of authentication methods supported by the provider.
+                      properties:
+                        assisted:
+                          $ref: '#/components/schemas/AuthenticationMethod'
+                        api_token:
+                          $ref: '#/components/schemas/AuthenticationMethod'
+                        oauth:
+                          $ref: '#/components/schemas/AuthenticationMethod'
+                        credential:
+                          $ref: '#/components/schemas/AuthenticationMethod'
```

Comment:
need to change back to array of objects with `type`

Reply:
Done!